scripts/import/laptops/update_visit_data

```diff
@@ -152,7 +153,7 @@ def connect_to_redcap():
 try:
     rc_summary = connect_to_redcap()
 except Exception, error:
-    sibis.logging(hashlib.sha1('update_summary_scores').hexdigest()[0:6],
+    sibis.logging(hashlib.sha1('update_visit_date').hexdigest()[0:6],
     "ERROR: Could not connect to redcap!",
     script = 'update_visit_date')
```

Comment:
there should not be a space `script='update_visit_date'`, the indentation should be aligned under `hashlib`